src/fmu/dataio/_model/fields.py

```diff
@@ -475,7 +471,7 @@ class FMUCaseAttributes(BaseModel):
     See :class:`Model`."""
 
 
-class FMUAttributes(FMUCaseAttributes):
+class FMU(FMUBase):
```

Comment:
A sneak changed attached in this

Reply:
> FMUBase is intended as a base class over FMUCase ...

As base class over `FMU`?

Reply:
Worded this confusingly... I just meant to say that naming it `FMUBase` was intended, not a typo of `FMUCase`

Reply:
We could also consider another module name than `fields.py`, since in oil industry we think of _fields_ as specific oil/gas fields?

Reply:
I debated on this one too, there is a lot of overlap. Other options are `_datamodel` over `_model`, and maybe `attributes.py` or `blocks.py` instead of `fields.py`. I guess the problem is that these things are trying to describe the datamodel, but the datamodel is describing the domain, so it's not clear which way to lean them